packages/apollos-data-prayer/src/prayer-menu-categories/schema.js

```diff
@@ -0,0 +1,17 @@
+import gql from 'graphql-tag';
+
+const prayerMenuCategorySchema = gql`
+  extend type Query {
+    prayerMenuCategories: [PrayerMenuCategory]
+  }
+  type PrayerMenuCategory implements Node {
+    id: ID!
+    key: String!
+    title: String!
+    subtitle: String
```

Comment:
@redreceipt Are these ever going to be anything other than Strings?

Reply:
not sure what else they would be

Reply:
@cdanwards what do you suggest?